edx_notifications/startup.py

```diff
@@ -26,4 +26,8 @@ def initialize():
     # notification types
     perform_type_registrations.send(sender=None)
 
+    # alert the application tiers that they should register their
+    # notification timers/callbacks
+    perform_timer_registrations.send(sender=None)
```

Comment:
If you use this signaling approach, it's an unfortunate thing in python that you will also have to put up at the top:

import digest_timers  # pylint: disable=unsed-import   (I think that's the pylint error override, see a similar example)

Otherwise, python will not load that digest_timer.py module before perform_timer_registrations.send() is called and the receiver that is defined there will not get called.